src/coreComponents/dataRepository/Wrapper.hpp

```diff
@@ -936,6 +946,8 @@ class Wrapper final : public WrapperBase
   /// address of m_data
   bool m_ownsData;
 
+  bool m_isClone;
```

Comment:
What is this `m_isClone` attribute for? It looks like it's an "ownership" flag?
But are we sure that all the clones do not own their data? Or that any new data can't be added to them?

Reply:
a cloned wrapper is different than ownership. We have wrappers that are clones of other wrappers, i.e. they have equal values of `Wrapper::m_data` (`m_data` is a pointer to the wrapped object). In many cases neither of the `Wrapper` will have `m_ownsData=true`. To avoid printing the same allocation in different `wrapper` objects I added this flag to use in a filter.

Reply:
I guess I could always save the addresses of the `Wrapper::m_data` pointer and not repeat them, but then that requires a hash of all `Wrapper::m_data`, and we won't get the original `Wrapper`...just the first one we come across....as a matter of fact, I do this to exclude shallow `Group` copies...which will suffer from this same issue.

Reply:
If you store the addresses of `m_data`, you'd be able not to print it twice (or indicate that it's "duplicated", which can be OK).
Is it a big concern not to know the original `wrapper`?
Is it possible that new `Wrappers` (with new unique data) get added to a `Group` with `m_isClone` set to true, and then the `wrapper` gets ignored?

Reply:
> If you store the addresses of `m_data`, you'd be able not to print it twice (or indicate that it's "duplicated", which can be OK).
> Is it a big concern not to know the original `wrapper`?

it is a matter of preference rather than a "big concern"

> Is it possible that new `Wrappers` (with new unique data) get added to a `Group` with `m_isClone` set to true, and then the `wrapper` gets ignored?

no.